### PR TITLE
Crl master

### PR DESCRIPTION
**Pull request by denniszollo**
Here as a branch to facilitate auto-building of master + CRL daemon on travis.  Still requires someone to press "rebuild" button on travis for the merge build, but better than having to constantly rebase.

### CODE REVIEW

File: package/piksi_ins/Config.in [hunk 1]
```diff
@@ -0,0 +1,7 @@
+config BR2_PACKAGE_PIKSI_INS
+	bool "piksi_ins"
```
Comment (silverjam) [inline package/piksi_ins/Config.in]: Use spaces to match indents of other lines.

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.4.0-develop-2018033016-17-g34b93a2
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=484184)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018033016-17-g34b93a2&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018033016-17-g34b93a2&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.4.0-develop-2018033016-17-g34b93a2
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.4.0-develop-2018033016-17-g34b93a2)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.4.0-develop-2018033016-17-g34b93a2/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.4.0-develop-2018033016-17-g34b93a2

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.4.0-develop-2018033016-19-g8bdae5d
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=486164)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018033016-19-g8bdae5d&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018033016-19-g8bdae5d&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.4.0-develop-2018033016-19-g8bdae5d
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.4.0-develop-2018033016-19-g8bdae5d)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.4.0-develop-2018033016-19-g8bdae5d/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.4.0-develop-2018033016-19-g8bdae5d

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.4.0-develop-2018061217-14-gb62aadb3
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=550992)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018061217-14-gb62aadb3&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018061217-14-gb62aadb3&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.4.0-develop-2018061217-14-gb62aadb3
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.4.0-develop-2018061217-14-gb62aadb3)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.4.0-develop-2018061217-14-gb62aadb3/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.4.0-develop-2018061217-14-gb62aadb3

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.4.0-develop-2018061217-14-gb62aadb3
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=550993)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018061217-14-gb62aadb3&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018061217-14-gb62aadb3&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.4.0-develop-2018061217-14-gb62aadb3
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.4.0-develop-2018061217-14-gb62aadb3)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.4.0-develop-2018061217-14-gb62aadb3/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.4.0-develop-2018061217-14-gb62aadb3

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.4.0-develop-2018061217-15-g50accc26
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=551017)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018061217-15-g50accc26&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018061217-15-g50accc26&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.4.0-develop-2018061217-15-g50accc26
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.4.0-develop-2018061217-15-g50accc26)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.4.0-develop-2018061217-15-g50accc26/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.4.0-develop-2018061217-15-g50accc26

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v1.4.0-develop-2018061217-15-g50accc26
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=551018)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018061217-15-g50accc26&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v1.4.0-develop-2018061217-15-g50accc26&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v1.4.0-develop-2018061217-15-g50accc26
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v1.4.0-develop-2018061217-15-g50accc26)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.4.0-develop-2018061217-15-g50accc26/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.4.0-develop-2018061217-15-g50accc26

**Pull request closed without merging**